### PR TITLE
Data Preservation with JSON

### PR DESCRIPTION
**Pull request by Shakir-Hussain12**
**For this PR, myself and @uchexm :**
- Created respective JSON files for `person`, `book`, and `rental` class & saved data to them using `File.write`.
- Added `Loader` class which retrieves data from the above using `File.read` (if they exist) & saves them to their respective arrays within `main.rb`.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -4,21 +4,29 @@
...
 class App
   puts "Welcome to school library app!\n\n"
   def initialize
     @books = []
     @people = []
     @rentals = []
+    loader = Loader.new
+    loader.load_people(@people)
+    loader.load_books(@books)
+    loader.load_rentals(@rentals)
```
Comment (mukhammadrizooff) [inline app.rb:19]: - [x] [REQUIRED] Good job so far! Kindly your `Exit` option has a error therefore I can't find `loaded` files, please fix it🙏🏼

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @TEAM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (nwakauc) [reply app.rb:19]: Aye Aye Captain 👏

**Review by DammyShittu — APPROVED** [review]
Hi @Shakir-Hussain12 and @uchexm,

## APPROVED 🟢

## Highlights
✔️ File storage for data

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**